contracts/Comptroller/Diamond/facets/SetterFacet.sol

```diff
@@ -576,6 +597,33 @@ contract SetterFacet is ISetterFacet, FacetBase {
         xvsVToken = xvsVToken_;
     }
 
+    /**
+     * @notice Set the liquidation incentive for a market
+     * @param vToken The market to set the liquidation incentive for
+     * @param newMaxLiquidationIncentive The new liquidation incentive, scaled by 1e18
+     * @return uint256 0=success, otherwise reverted
+     */
+    function _setMarketMaxLiquidationIncentive(
+        address vToken,
+        uint256 newMaxLiquidationIncentive
+    )
+        external
+        compareValue(markets[address(vToken)].maxLiquidationIncentiveMantissa, newMaxLiquidationIncentive)
+        returns (uint256)
+    {
+        ensureAllowed("_setMarketMaxLiquidationIncentive(address,uint256)");
```

Comment:
We're removing an underscore in _setCollateralFactor (name and access controls), but on the other hand we're adding two more functions that start with an underscore, this looks inconsistent. Let's maybe treat underscores as legacy and use non-prefixed names for new methods? AFAIU [this PR](https://github.com/VenusProtocol/venus-protocol/pull/548) was in the direction of deprecating underscores

Reply:
[ab6d0ae](https://github.com/VenusProtocol/venus-protocol/pull/604/commits/ab6d0ae46348f8f75f1385cb036cddab880b8457)